src/open_inwoner/components/templatetags/filter_tags.py

```diff
@@ -4,12 +4,12 @@
 
 
 @register.inclusion_tag("components/Filter/Filter.html")
-def filter(field, **kwargs):
+def filters(field, **kwargs):
```

Comment:
This is just a cosmetic change so it doesn't shadow the global `filter` function.